src/winter_onboarding_2021/fleet_management_service/core.clj

```diff
@@ -0,0 +1,29 @@
+(ns winter-onboarding-2021.fleet-management-service.core
+  (:require [ring.adapter.jetty :as raj]
+            [winter-onboarding-2021.fleet-management-service.routes :as routes])
+  (:gen-class))
+
+(defonce server (atom nil))
+
+(defn hello-world [request]
+  {:status 200
+   :headers {"Content-Type" "text/html"}
+   :body "Hello from my API!"})
+
+(defn start-server []
+  (reset! server (raj/run-jetty routes/ring-handler
```

Comment:
`raj/run-jetty` call can be moved into the next line

Reply:
Done